app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
 <h1>Specific user</h1>
+<div class='user-class'>
+  <span class='title'> <%= @user.name %> </span> <span class='number-of-posts'>Number of posts: <%= @user.post_counter %></span>
+</div>
+<div class='bio-class'>
+  <h4>Bio</h4>
+  <%= @user.bio %>
+</div>
+<div class='posts-container'>
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div class='user-posts'>
+      <h2>Post #<%= index+1 %>
+      <p class='post'> <%= post.text %></p>
```

Comment:
- [ ] There is an error raised here, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/223888759-83b7145e-d3ed-40f7-a087-71df653e139d.png)